pkg/topology/kademlia/kademlia.go

```diff
@@ -668,28 +672,31 @@ func (k *Kad) connect(ctx context.Context, peer swarm.Address, ma ma.Multiaddr,
 func (k *Kad) Announce(ctx context.Context, peer swarm.Address) error {
 	addrs := []swarm.Address{}
 
-	_ = k.connectedPeers.EachBinRev(func(connectedPeer swarm.Address, _ uint8) (bool, bool, error) {
-		if connectedPeer.Equal(peer) {
-			return false, false, nil
+	//r := rand.New(securerand.NewSource())
+
+	for bin := uint8(0); bin < swarm.MaxBins; bin++ {
```

Comment:
r: @acud is there a reason why `swarm.MaxBins` is a typed constant? If not, I'll remove the type and change the loop: `for bin := 0; bin < swarm.MaxBins; bin++ {...` (also in the affected code in the `pkg/swarm/proximity.go` file), more on this topic: https://blog.golang.org/constants